### ISSUE TITLE
Pass on the control that fires the event

### ISSUE THREAD

**Issue by ka7eh** [Leaflet/Leaflet.draw#750]
I have two draw controls and I'd like to take different actions on `draw:created` and `draw:edited` events of each. Is there anyway to tell which control has fired the event? So far I haven't found any straightforward way of doing so.
Thanks.

**Comment by jbccollins** [comment]
Yeah I really need this feature. My use case is that I want two sets of controls: One set for Green shapes and one set for Red shapes that are used to include and exclude regions in a search.

Will update if I find a solution....
